src/components/TodoItem.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onRemove: (userId: number) => void;
+  onDeleting: boolean;
```

Comment:
Change the name of the 'onDeleting' prop to 'isDeleting' to better represent its boolean nature.